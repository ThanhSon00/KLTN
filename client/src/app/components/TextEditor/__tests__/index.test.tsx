import * as React from 'react';
import { render } from '@testing-library/react';

import { TextEditor } from '..';

describe('<TextEditor  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<TextEditor />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
