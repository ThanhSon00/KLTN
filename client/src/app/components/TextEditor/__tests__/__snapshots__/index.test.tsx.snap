// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextEditor  /> should match snapshot 1`] = `
<textarea
  id="tiny-react_89735591611710829088234"
  style="visibility: hidden;"
/>
`;
